manifests/exporters/node-exporter-svc.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
     app: node-exporter
   annotations:
     prometheus.io/scrape: 'true'
+    prometheus/job_name: node
```

Comment:
If you envision the operator itself supporting such annotations in the future, we could go for `monitoring.coreos.io/prometheus_job` or so.